packages/components/src/theme/panel.scss

```diff
@@ -0,0 +1,41 @@
+.panel-content {
+    @include panel-variant($panel-content-border, $panel-content-text, $panel-content-heading-bg, $panel-content-border);
```

Comment:
I'm curious about this `panel-variant`. This is a built in bootstrap thing?

Reply:
Yes. See https://github.com/twbs/bootstrap-sass/blob/b34765d8a6aa775816c59012b2d6b30c4c66a8e9/assets/stylesheets/bootstrap/mixins/_panels.scss

Reply:
Yes. See https://github.com/twbs/bootstrap-sass/blob/b34765d8a6aa775816c59012b2d6b30c4c66a8e9/assets/stylesheets/bootstrap/mixins/_panels.scss